love/src/components/MainTel/M1M3TS/Info/Info.jsx

```diff
@@ -0,0 +1,192 @@
+    /** Array for the identify of the position in array with an index */
+    sensorReferenceId: PropTypes.arrayOf(PropTypes.number),
+    /** Id of sensor selected */
+    selectedSensor: PropTypes.number,
+    /** True if this fan unit is enabled. */
+    // enabled: PropTypes.arrayOf(PropTypes.bool),
+    /** Thermal status response data. Absolute temperature. */
+    absoluteTemperature: PropTypes.arrayOf(PropTypes.number),
+    /** Thermal status response data.  Differential temperature. */
+    differentialTemperature: PropTypes.arrayOf(PropTypes.number),
+    /** Thermal status response data.  Measured fan RPM-0 to 2550 RPM. */
+    fanRPM: PropTypes.arrayOf(PropTypes.number),
+    /** Object with boolean arrays for indicates of warning. */
+    thermalWarnings: PropTypes.object,
+  };
+
+  static defaultProps = {
+    sensorReferenceId: [],
+    selectedSensor: undefined,
+    // enabled: [],
+    absoluteTemperature: [],
+    differentialTemperature: [],
+    fanRPM: [],
+    thermalWarnings: {},
+  };
+
+  getThermalWarnings = (sensorIndex) => {
+    const { thermalWarnings } = this.props;
+
+    const warnings = Object.keys(thermalWarnings)
+      .map((key, _) => ({
+        name: key,
+        value: thermalWarnings[key][sensorIndex],
+        msg: MessagesWarningM1M3ThermalSystem[key],
+      }))
+      .filter((warning) => warning.value === true);
+
+    return warnings;
+  };
+
+  getSensor = (id) => {
+    if (id === undefined) return { id: 'None', value: 'None', state: CSCDetail.states[0] };
+
+    const {
+      sensorReferenceId,
+      ilcFCU,
+      // enabled,
+      absoluteTemperature,
+      differentialTemperature,
+      fanRPM,
+      fanBreaker,
+      heaterDisabled,
+    } = this.props;
+
+    const fcuIndex = M1M3TSFanCoilPositions.findIndex((fcu) => fcu.id === id);
+    const sensorIndex = sensorReferenceId.indexOf(fcuIndex);
+
+    // const enabledData = enabled[sensorIndex];
+    // const enabledState = m1m3tsEnabledStateMap[enabledData];
+
+    const ilcData = ilcFCU[sensorIndex];
+    const ilcState = m1m3tsActuatorILCStateMap[ilcData];
+
+    const fanBreakerData = fanBreaker[sensorIndex];
+    const fanBreakerState = mtm1m3tsFanBreakerStateMap[fanBreakerData];
+
+    const heaterDisabledData = heaterDisabled[sensorIndex];
+    const heaterDisabledState = mtm1m3HeaterDisabledStateMap[heaterDisabledData];
+
+    const warnings = this.getThermalWarnings(sensorIndex);
+
+    const sensor = {
+      index: sensorIndex,
+      name: `FCU${String(id).padStart(2, '0')}`,
+      absoluteTemperature: absoluteTemperature[sensorIndex] ?? 0,
+      differentialTemperature: differentialTemperature[sensorIndex] ?? 0,
+      fanRPM: fanRPM[sensorIndex] ?? 0,
+      fanBreaker: fanBreaker[sensorIndex] ?? false,
+      heaterDisabled: heaterDisabled[sensorIndex] ?? false,
+      state: {
+        ilc: {
+          name: ilcState,
+          status: m1m3tsILCStateToStyle[ilcState],
+        },
+        // enabled: {
```

Comment:
Will this variable be used? If not, maybe it should be removed.

Reply:
Same as stated above.